docs/src/opf/acp.md

```diff
@@ -1,3 +1,149 @@
 # AC-OPF
 
-See [`ACPPowerModel`](https://lanl-ansi.github.io/PowerModels.jl/stable/formulation-details/#PowerModels.ACPPowerModel) formulation in [`PowerModels.jl`](https://lanl-ansi.github.io/PowerModels.jl/stable/).
\ No newline at end of file
+
+## Mathematical Formulation
+
+The ACOPF model considered in OPFGenerator is presented below.
+
+```math
+\begin{align}
+    \min_{\PG, \QG, \PF, \QF, \PT, \QT, \VM, \VA} \quad
+    & \sum_{i \in \NODES} \sum_{j \in \GENERATORS_{i}} c_j \PG_j \label{model:acopf:obj} \\
+    \text{s.t.} \quad \quad \quad
+    & \sum_{j\in\GENERATORS_i}\PG_j - \sum_{j\in\LOADS_i}\PD_j - \GS_i \VM_i^2 = \sum_{e \in \mathcal{E}_{i}}  \PF_{e} + \sum_{e \in \mathcal{E}^R_{i}} \PT_{e}
```

Comment:
This objective does not include constant terms, which we do include in the code

Reply:
Added a $+c_0$